backend/main/src/Group/UseCases/CreateContainerUseCase/CreateContainerUseCase.ts

```diff
@@ -0,0 +1,50 @@
+import { IContainerRepository } from "src/Group/Domain/IContainerRepository";
+import { Container, ContainerId } from "src/Group/Domain/Entities/Container";
+import { IUseCase } from "src/Shared";
+import { Err, Ok, Result } from "result-ts-type";
+import {
+	CreateContainerUseCaseErrorType,
+	ICreateContainerUseCase,
+} from "src/Group/UseCases/CreateContainerUseCase/ICreateContainerUseCase";
+import { IContainerDto, containerDtoMapper } from "src/Group/Dtos/ContainerDto";
+
+/**
+ * Create a new container. You can call this use case without a container name, and then the new container's name will be the default name.
+ * @throws: When you try to create a new container that already existing, the error will throw an error.
```

Comment:
I could not understand when this error would be thrown. Additionally, that error can be checked easily,  I think it's better to handle it explicitly somehow using the result type.